client/src/components/Messaging/Message.js

```diff
@@ -45,9 +45,9 @@ export const Message = ({
           {timestamp && (
             <span className={styles.timestamp}>
               {(owner === true ? 'sent at ' : 'received at ') +
-                new Date(timestamp).getHours() +
+                (time.getHours() < 10 ? '0' + time.getHours() : time.getHours()) +
```

Comment:
What about just using `new Date().toLocaleTimeString()`

Reply:
Then seconds will also be displayed if that is not required then seconds will have to be removed manually.

Reply:
`new Date().toLocaleTimeString([], {hour: '2-digit', minute:'2-digit'})`